src/app/assinante/component/assinante.component.ts

```diff
@@ -1,35 +1,78 @@
 import { Component, OnInit } from '@angular/core';
+import { response } from 'express';
+import { Artista } from 'src/app/artista/service/artista';
+import { ArtistaService } from 'src/app/artista/service/artista.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
import { ArtistaService } from "src/app/artista/service/artista.service";
```